### ISSUE TITLE
Migrate to Swift 3

### ISSUE THREAD

**Issue by damienrambout** [toygard/youtube-parser#19]
Hi,

What is the ETA for a Swift 3 version of **youtube-parser**?

Cheers.

**Comment by lennet** [comment]
I submitted a PR (#21) with Swift 3 support

**Comment by tonifirnandes** [comment]
Yes please update this library to support Xcode 8.3.3 Swfit 3 version thanks.
